docs/streams/developer-guide.html

```diff
@@ -18,7 +18,7 @@
 <script><!--#include virtual="../js/templateData.js" --></script>
 
 <script id="content-template" type="text/x-handlebars-template">
-    <h1>Developer Guide</h1>
+    <h1>Developer Manual</h1>
```

Comment:
@miguno I am going to merge as is since it is what the kafka-site shows. If you feel strong let's re-discuss it in a follow-up PR.